image/cli/mascli/functions/gitops_utils

```diff
@@ -282,11 +283,9 @@ function check_argo_app_synced() {
       echo "Sync Status is $SYNC_STATUS, Waiting 30s before checking status again"
       ((COUNT++))
       if ! (( $COUNT % 10 )) ; then
-        echo "Restart application-controller and try resync"
-        oc rollout restart statefulset/openshift-gitops-application-controller -n openshift-gitops
-        echo 'Waiting for rollout of application-controller to complete'
-        oc rollout status statefulset/openshift-gitops-application-controller -n openshift-gitops
-
+        if [ -n "${NAMESPACE}" ]; then
+          oc adm policy add-role-to-user admin system:serviceaccount:openshift-gitops:openshift-gitops-argocd-application-controller -n ${NAMESPACE}
```

Comment:
Can we add a comment just to say why we are going this. Just so when we look back 6 months we can understand why we added this call

Reply:
Added the comment -
# Fix for Issue : Cannot sync application in OpenShift GitOps. https://access.redhat.com/solutions/6331341